### DISCUSSION TITLE
ncmpcpp, music player,   not work after upgrading.  icu4c isuues.

### DISCUSSION THREAD

**Discussion by wan2355** [Homebrew/discussions#1669] [Everyday usage]
I am using mpd and ncmcpp on my Mac, OSX 10.14.6. , and Linux (manjaro). 
After upgrading Mac homebrew a few days ago,  ncmcpp dose not work.

```
$ ncmpcpp

dyld: Library not loaded: /usr/local/opt/icu4c/lib/libicui18n.68.dylib
  Referenced from: /usr/local/bin/ncmpcpp
```

I switch icu4c to  icu4c.68.2 by symbolik link then another error come up.
I know the way to go back to old version by using git check, but it is a little bit bother... 

```
$ cd /usr/local/opt
$ mv icu4c icu4c69.1
$ ln -s   /usr/local/Cellar/icu4c/68.2  /ust/local/opt/icu4c

$ ncmpcpp

dyld: Library not loaded: /usr/local/opt/icu4c/lib/libicudata.69.dylib
  Referenced from: /usr/local/opt/boost/lib/libboost_locale-mt.dylib
  Reason: image not found
``` 

I reinstall boost?, but same errors still appear. 
I look for solution on the net but I can not find it, and I find that icu4c causes issues on OSX.
  
I use mpd and ncmcpp on my Linux PC but I have no-issues.

Dose anyone help me ?

**Comment by SMillerDev** [comment]
> I switch icu4c to icu4c.68.2 by symbolik link then another error come up.

This is a very bad idea, you're breaking a lot of homebrew assumptions here. The solution to `dyld: Library not loaded: /usr/local/opt/icu4c/lib/libicui18n.68.dylib` is to update `ncmpcpp` so it knows about the libicu4c update.

**Comment by wan2355** [comment]
Thanks.

I have updated ncmcpcc as you say.

Then ncmcppp can not connect mpd....

```
Timeout while connecting
```

I update mpd , then ncmcpp can connect mpd.
